feature/amazon/src/main/java/com/revenuecat/purchases/amazon/AmazonBilling.kt

```diff
@@ -75,56 +77,15 @@ internal class AmazonBilling constructor(
         onReceivePurchaseHistory: (List<PurchaseDetails>) -> Unit,
         onReceivePurchaseHistoryError: PurchasesErrorCallback
     ) {
-        purchaseUpdatesHandler.queryPurchases(
-            onSuccess = { receipts, userData ->
-                if (receipts.isEmpty()) {
-                    onReceivePurchaseHistory(emptyList())
-                    return@queryPurchases
-                }
-                getMissingSkusForReceipts(
-                    amazonUserID = userData.userId,
-                    receipts
-                ) { tokensToSkusMap, errors ->
-                    logErrorsIfAny(errors)
-
-                    if (tokensToSkusMap.isEmpty()) {
-                        val error = PurchasesError(
-                            PurchasesErrorCode.InvalidReceiptError,
-                            AmazonStrings.ERROR_FETCHING_PURCHASE_HISTORY_ALL_RECEIPTS_INVALID
-                        )
-                        onReceivePurchaseHistoryError(error)
-                        return@getMissingSkusForReceipts
-                    }
-
-                    val purchaseHistoryRecordWrappers =
-                        receipts.toPurchaseHistoryRecordWrappers(tokensToSkusMap, userData)
-
-                    onReceivePurchaseHistory(purchaseHistoryRecordWrappers)
-                }
```

Comment:
I realized the code is the same as `queryPurchases` but returning a list instead of a map